src/components/value-list/value-list.tsx

```diff
@@ -279,7 +278,9 @@ export class ValueList<
 
     const { el } = this;
     const nextIndex = moveItemIndex(this, item, event.key === "ArrowUp" ? "up" : "down");
-
+    this.updateLiveText(
+      `New Position ${nextIndex + 1} of ${this.getItems().length}, press space to confirm `
```

Comment:
This needs to be an `intl` property as well so we can have it localized. It might need to be a HTML template so that the numbers can be inserted.